src/components/Banner.tsx

```diff
@@ -71,36 +57,36 @@ function Banner(props) {
                             styles.borderRadiusNormal,
                             shouldHighlight ? styles.activeComponentBG : styles.hoveredComponentBG,
                             styles.breakAll,
-                            ...props.containerStyles,
+                            containerStyles,
                         ]}
                     >
                         <View style={[styles.flexRow, styles.flexGrow1, styles.mw100, styles.alignItemsCenter]}>
-                            {props.shouldShowIcon && (
+                            {shouldShowIcon && (
                                 <View style={[styles.mr3]}>
                                     <Icon
                                         src={Expensicons.Exclamation}
                                         fill={StyleUtils.getIconFillColor(getButtonState(shouldHighlight))}
                                     />
                                 </View>
                             )}
-                            {props.shouldRenderHTML ? (
-                                <RenderHTML html={props.text} />
+                            {shouldRenderHTML ? (
+                                <RenderHTML html={text} />
                             ) : (
                                 <Text
-                                    style={[...props.textStyles]}
-                                    onPress={props.onPress}
+                                    style={textStyles}
+                                    onPress={onPress}
                                     suppressHighlighting
                                 >
-                                    {props.text}
+                                    {text}
                                 </Text>
                             )}
                         </View>
-                        {props.shouldShowCloseButton && (
-                            <Tooltip text={props.translate('common.close')}>
+                        {shouldShowCloseButton && !!onClose && (
```

Comment:
This will probably create a regression [here](https://github.com/Expensify/App/blob/main/src/components/ArchivedReportFooter.js#L76) as `onClose`/`onPress` props are optional.

Reply:
@fabioh8010 Since `onClose` is passed together with `shouldShowCloseButton`, and in your example  `shouldShowCloseButton` isn't passed and will be `false` by default, I think it should be ok.

Reply:
Makes sense, thanks!